src/main/resources/migration/V0_3__Create_form_table.sql

```diff
@@ -1,11 +1,11 @@
-CREATE DATABASE form(
-    id varchar(250) primary key,
+CREATE TABLE IF NOT EXIST "form"(
+    id varchar(255) primary key,
     title varchar(250) not null,
     description varchar(250),
     visibility varchar (250),
-    created_at datetime,
-    updated_at datetime,
-    open_at datetime,
-    close_at datetime,
-    id varchar(250) REFERENCES user(id)
+    created_at timestamp,
+    updated_at timestamp,
+    open_at timestamp,
+    close_at timestamp,
```

Comment:
timestamp -> timestampz (to say with time zone, sorry if I said without timezone but in fact it should be **timestampz**)